systers_portal/templates/500.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block title %} - Server Error{% endblock %}
+
+{% block content %}
+  
+  <div class="jumbotron jumbotron-custom">
+    <h1>500</h1>
+    <h1>Server Error</h1>
```

Comment:
r/Server Error/Internal Server Error plus what I said above.

Reply:
"r/Server Error/Internal Server Error" 
Are you talking about the error message here?